src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/NetCorePackageReferenceProjectProvider.cs

```diff
@@ -111,7 +111,8 @@ public NetCorePackageReferenceProjectProvider(IProjectSystemCache projectSystemC
                 _projectSystemCache,
                 unconfiguredProject,
                 projectServices,
-                vsProject.ProjectId);
+                vsProject.ProjectId,
+                targetFramework);
```

Comment:
Unfortunately this is not quite correct. 

The `TargetFramework` property is something that we shouldn't be parsing. Instead NuGet relies on the `TargetPlatformMoniker` and `TargetFrameworkMoniker`. 

For .NET Core projects it's the package spec that contains the target frameworks, and that comes through nomination at a later point. 

At this point we *do not* know the target framework.

Reply:
Changed this to get the target framework from the package spec.